src/parse.js

```diff
@@ -126,8 +126,23 @@ export const getTraverser = (cb = noop, opts = {}) => {
         state.opts = {
           ...state.opts,
           ...opts,
-          filename: state.file ? state.file.opts.filename : opts.filename,
         };
+
+        let filename = state.file ? state.file.opts.filename : opts.filename;
+
+        if (filename) {
+          switch (state.opts.filename) {
+            case 'relative':
+              filename = filename.replace(process.cwd(), '');
```

Comment:
Sorry, I see no point in having absolute paths. They are private, custom to a singel machine and unreliable/subject to change whenever a project is moved around. It might even be a security risk to expose full paths.

Implement the `filename.replace(process.cwd(), '');` solution and I think that'll do! Thank you for all the work, @iszak.